.github/workflows/zulip_emoji_merge_delegate.yaml

```
@@ -0,0 +1,83 @@
+name: Zulip Emoji Merge Delegate
+
+on:
+  schedule:
+    - cron: '0 * * * *'  # Runs every hour
+
+jobs:
+  zulip-emoji-merge-delegate:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install zulip
+
+    - name: Run Zulip Emoji Merge Delegate Script
+      env:
+        ZULIP_API_KEY: ${{ secrets.ZULIP_API_KEY }}
+        ZULIP_EMAIL: ${{ secrets.ZULIP_EMAIL }}
+        ZULIP_SITE: https://leanprover.zulipchat.com
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        python - <<EOF
+        import zulip
+        import requests
+        from datetime import datetime, timedelta
+
+        # Initialize Zulip client
+        client = zulip.Client(config_file=None)
+        client.set_api_key("${{ secrets.ZULIP_API_KEY }}")
+        client.set_email("${{ secrets.ZULIP_EMAIL }}")
+        client.set_site("https://leanprover.zulipchat.com")
+
+        # Calculate the time range
+        now = datetime.utcnow()
+        since = now - timedelta(minutes=75)
+
+        # Fetch messages from the past 1h15
+        response = client.get_messages({
+            "anchor": "newest",
+            "num_before": 0,
+            "num_after": 1000,
+            "narrow": [
+                {"operator": "stream", "operand": "general"},
+                {"operator": "after", "operand": since.isoformat()}
+            ]
+        })
+
+        messages = response['messages']
+        pr_pattern = re.compile(r'#(\d{1,5})')
```

Not sure if this is the best way to do this, but presumably we'll want to filter out linkifiers pointing to other repos.
```suggestion
        pr_pattern = re.compile(r'(^|\s)#(\d{1,5})')
```
There's also the `!4#XXXXX` linkifier, but I think only the triage bot uses it (and only in the titles of threads).